src/components/flowchart/draw.js

```diff
@@ -136,6 +150,16 @@ export const drawNodes = function (changed) {
     focusMode,
     hoveredFocusMode,
   } = this.props;
+  const { filteredPipelineState } = this.state;
+
+  const fromToFilteredPipeline = createNodeStateMap(nodes, [
```

Comment:
these names are a bit confusing. I cannot come up with better names but below is the best. 
I am also ok if we stick to these names  but lets leave a comment on top to define what they do

```
// Map the state of nodes selected by the user (from and to nodes)
const selectedNodesState = createNodeStateMap(nodes, [
  filteredPipelineState.from,
  filteredPipelineState.to,
]);

// Map the state of nodes that fall within the range between the selected nodes
const nodesInRangeState = createNodeStateMap(
  nodes,
  filteredPipelineState.range
);
```

Reply:
let me have a think, I'll come back to you on this